template/template.go

```diff
@@ -30,10 +30,27 @@ import (
 
 	"github.com/prometheus/common/model"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/prometheus/util/strutil"
 )
 
+var (
+	textTemplateExpansionFailures = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_text_template_expansion_failures_total",
```

Comment:
template_text

This is the template subsystem.

Reply:
Sorry, because of the import statement, `text_template "text/template"`, I renamed to this name. I'll rename to `prometheus_template_text_expansion_failures_total` and `prometheus_template_text_expansions_total`.